spec/services/ccms/parsers/case_add_status_response_parser_spec.rb

```diff
@@ -8,36 +8,64 @@ module Parsers
           let(:response_xml) { ccms_data_from_file 'case_add_status_response.xml' }
           let(:expected_tx_id) { '20190301030405123456' }
 
-          it 'extracts the status' do
-            parser = described_class.new(expected_tx_id, response_xml)
-            expect(parser.success?).to eq true
-          end
+          context 'normal success response' do
+            let(:response_xml) { ccms_data_from_file 'case_add_status_response.xml' }
```

Comment:
Is this needed as well as line 8?

Reply:
no you're right.  line 8 can go.  have removed it